src/Issue/Version.php

```diff
@@ -11,7 +11,7 @@ class Version implements \JsonSerializable
     public string $id;
 
     // Version name: ex: 4.2.3
-    public ?string $name;
+    public string $name;
```

Comment:
this causes an exception when unserializing, because name is not passed to the constuctor. Probably a bug in jsonmapper though
```
#0 [internal function]: JiraRestApi\Issue\Version->__construct()
#1 vendor/netresearch/jsonmapper/src/JsonMapper.php(685): ReflectionClass->newInstance()
#2 vendor/netresearch/jsonmapper/src/JsonMapper.php(472): JsonMapper->createInstance()
#3 vendor/lesstif/php-jira-rest-client/src/Project/ProjectService.php(270): JsonMapper->mapArray()
```

Reply:
https://github.com/lesstif/php-jira-rest-client/pull/521
already fixed but waiting for owner to respond :)